Use the Python environement from EB in the Guroby Python shell
autho: Alex Domingo (Vrije Universiteit Brussel)
--- a/bin/gurobi.sh	2020-01-29 00:56:47.000000000 +0100
+++ b/bin/gurobi.sh	2021-01-11 16:07:28.180329000 +0100
@@ -7,11 +7,6 @@
   echo
 fi
 
-PATH=$GUROBI_HOME/bin:$PATH;export PATH
-LD_LIBRARY_PATH=$GUROBI_HOME/lib:$LD_LIBRARY_PATH;export LD_LIBRARY_PATH
-PYTHONHOME=$GUROBI_HOME;export PYTHONHOME
-PYTHONPATH=$GUROBI_HOME:$PYTHONPATH;export PYTHONPATH
+PYTHONSTARTUP=$EBROOTGUROBI/lib/gurobi.py;export PYTHONSTARTUP
 
-PYTHONSTARTUP=$PYTHONHOME/lib/gurobi.py;export PYTHONSTARTUP
-
-$PYTHONHOME/bin/python3.7 "$@"
+$EBROOTPYTHON/bin/python "$@"
